packages/code-studio/vite.config.ts

```diff
@@ -40,6 +40,13 @@ export default defineConfig(({ mode }) => {
       target: `http://localhost:${port}/src/styleguide/index.html`,
       rewrite: () => '',
     },
+    // proxy the websocket, allows tunneling to work with a single port
+    '/': {
```

Comment:
Can't comment b/c it's not a changed line, but in the array in the block below this, you can remove `env.VITE_CORE_API_URL` if we don't have the env variable and don't need to proxy http requests to it

Reply:
No, we still need to proxy that so that we can get the jsapi files.

Reply:
No, we still need to proxy that so that we can get the jsapi files.